views/components/table.html

```diff
@@ -0,0 +1,22 @@
+{% macro table(data) %}
+<table class="base--table">
+  <caption class="base--caption">{{data.name}}</caption>
+  <thead class="base--thead">
+    <tr class="base--tr">
+      {% for header in data.headers %}
+      <th class="base--th">{{header}}</th>
+      {% endfor %}
+    </tr>
+  </thead>
+  <tbody class="base--tbody">
+    {% for rows in data.body %}
+    <tr class="base--tr" itemprop="dataset" itemscope itemtype="https://schema.org/Dataset">
```

Comment:
This was an error on my part. This should not have its own `itemscope` or an `itemprop` or `itemtype`